doc/classes/AcceptDialog.xml

```diff
@@ -55,6 +57,15 @@
 				Registers a [LineEdit] in the dialog. When the enter key is pressed, the dialog will be accepted.
 			</description>
 		</method>
+		<method name="remove_button">
+			<return type="void">
+			</return>
+			<argument index="0" name="button" type="Control">
+			</argument>
+			<description>
+				Removes the [code]button[/code] from the dialog. Does NOT free the [code]button[/code]. The [code]button[/code] must be a [Button] added with [method add_button] or [method add_cancel_button] method. After removal, pressing the [code]button[/code] will no longer emit this dialog's [signal custom_action] or [signal cancelled] signals.
```

Comment:
Seems dangerous to me to remove a button with no way to get a handle to it. That's going to be a memory leak. Either it should free it, or return it.

Reply:
You're passing it as an argument, you already have a reference to it.

Reply:
For example [removing stuff from the EditorPlugin](https://docs.godotengine.org/en/3.3/classes/class_editorplugin.html#class-editorplugin-method-remove-control-from-bottom-panel) has similar API.

Reply:
Ah right, I assumed the argument was an index.